packages/eslint-plugin/package.json

```diff
@@ -61,17 +61,17 @@
     "typecheck": "tsc -p tsconfig.json --noEmit"
   },
   "dependencies": {
-    "@eslint-community/regexpp": "^4.5.1",
+    "@eslint-community/regexpp": "^4.10.0",
     "@typescript-eslint/scope-manager": "7.5.0",
     "@typescript-eslint/type-utils": "7.5.0",
     "@typescript-eslint/utils": "7.5.0",
     "@typescript-eslint/visitor-keys": "7.5.0",
     "debug": "^4.3.4",
     "graphemer": "^1.4.0",
-    "ignore": "^5.2.4",
+    "ignore": "^5.3.1",
     "natural-compare": "^1.4.0",
-    "semver": "^7.5.4",
-    "ts-api-utils": "^1.0.1"
+    "semver": "^7.6.0",
+    "ts-api-utils": "^1.3.0"
```

Comment:
There's something weird about `1.3.0` of `ts-api-utils`. Source code of this version does not exist on GitHub.
Also `npm` flags it when executing `npm audit` (see https://github.com/JoshuaKGoldberg/ts-api-utils/issues/403).
Given the recent events with the XZ Utils backdoor, I would advise to pin this dependency to the last known version (`1.2.1`) to avoid any risks.

Reply:
@JoshuaKGoldberg is a maintainer of this project so can respond

Reply:
Though looking at it - there is provenance data for v1.3.0
https://www.npmjs.com/package/ts-api-utils/v/1.3.0#provenance
So it's verified as being published by the github action.

Reply:
@bradzacher And this is the linked GitHub Actions run: https://github.com/JoshuaKGoldberg/ts-api-utils/actions/runs/8214871649/job/22467737771.

This again looks a bit suspicious...
```
WARNING Unable to verify if user joshuakgoldberg is a collaborator for ts-api-utils.
```

Unsure about the rollback of pushing the tag below. But looks a bit a bit weird at least as well.

Reply:
I'm under the impression it was a transient npm issue, as mentioned in the ts-api-utils issue. I manually verified the code published matched what we expected from source. I'd encourage you to do the same if you're nervous about the release.

Reply:
Oh, and this isn't the only package that had the same odd issue: https://github.com/nodemailer/nodemailer/issues/1634